web-ng/src/app/components/layer-list-item/layer-list-item.component.html

```diff
@@ -32,7 +31,11 @@ <h3 matLine>
         <mat-icon>more_vert</mat-icon>
       </button>
       <mat-menu #layerMenu="matMenu">
-        <button mat-menu-item (click)="onCustomizeLayer()">
+        <button
+          mat-menu-item
+          (click)="onCustomizeLayer()"
+          *ngIf="canViewCustomizeLayer"
```

Comment:
Sorry I forgot to mention in previous review - please `canViewCustomize... ` might be a bit confusing - perhaps simply `canCustomizeLayer` would be clearer here.